### PR TITLE
Created stats controller

### PR DESCRIPTION
**Pull request by Quickzand**

### CODE REVIEW

**Comment by sethlenhof** [comment]
Are we sure this is what we want to do?

File: js/statsController.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var stats = {
+    strength: 0,
+    dexterity: 0,
+    constitution: 0,
+    intelligence: 0,
+    wisdom: 0,
+    charisma: 0,
+    hp: 0,
```
Comment (sethlenhof) [inline js/statsController.js:8]: remove comma

**Comment by maximusmith529** [comment]
lgtm 
https://tenor.com/view/bill-o-reilly-ship-it-lgtm-bill-do-it-live-gif-16059101

**Review by sethlenhof — APPROVED** [review]

**Review by maximusmith529 — APPROVED** [review]
lgtm
[image](https://tenor.com/view/bill-o-reilly-ship-it-lgtm-bill-do-it-live-gif-16059101)

**Pull request merged**